### ISSUE TITLE
Error requesting CHAOS-Static model in long request

### ISSUE THREAD

**Issue by smithara** [ESA-VirES/VirES-Server#241]
I'm hitting an error with a request that used to work before

> WPSError: WPS Request Failed! Reason: NoApplicableCode [ValueError]: setting an array element with a sequence. The requested array has an inhomogeneous shape after 1 dimensions. The detected shape was (2,) + inhomogeneous part.

```python
from viresclient import SwarmRequest

request = SwarmRequest()
request.set_collection("SW_OPER_MAGA_LR_1B")
request.set_products(
    measurements=["B_NEC"],
    models=["CHAOS-Static"],
    sampling_step="PT120S"
)
request.set_range_filter("Flags_F", 0, 1)
request.set_range_filter("Kp", 0, 3)
data = request.get_between("2019-01-01", "2020-01-01")
```

This request fails about 60% the way through, and seems to be specifically when `CHAOS-Static` is included.

The request works when using the other parts of the model, `["CHAOS-Core", "CHAOS-MMA-Primary", "CHAOS-MMA-Secondary"]`

**Comment by smithara** [comment]
I looked a bit more and it occurs specifically on "2019-09-01"

i.e. for `data = request.get_between("2019-09-01", "2019-09-02")`

**Comment by pacesm** [comment]
OK, there is definitely something not right.

This is the actual server error ...

```
  File ".../eoxserver/services/views.py", line 78, in ows
    result = handler.handle(request)
             ^^^^^^^^^^^^^^^^^^^^^^^
  File ".../eoxserver/services/ows/wps/v10/execute.py", line 172, in handle
    outputs = process.execute(**inputs)
              ^^^^^^^^^^^^^^^^^^^^^^^^^
  File ".../vires/processes/fetch_data.py", line 538, in execute
    for label, dataset in _generate_data_():
  File ".../vires/processes/fetch_data.py", line 470, in _generate_data_
    slave.interpolate(
  File ".../vires/processes/util/time_series/base_product.py", line 112, in interpolate
    dataset = self._subset_times(times, subset_variables, cdf_type)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File ".../vires/processes/util/time_series/cached_model.py", line 136, in _subset_times
    return self._get_empty_dataset(variables)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File ".../vires/processes/util/time_series/cached_model.py", line 358, in _get_empty_dataset
    dataset.set(self.time_variable, times, self.TIMESTAMP_TYPE, {})
  File ".../vires/dataset.py", line 61, in set
    data = array(data, copy=False)
           ^^^^^^^^^^^^^^^^^^^^^^^

ValueError: setting an array element with a sequence. The requested array has an inhomogeneous shape after 1 dimensions. The detected shape was (2,) + inhomogeneous part.```

**Comment by pacesm** [comment]
The issue is being investigated.

**Comment by pacesm** [comment]
Fixed in 0aa4b3c77c85de2a01024f7f68102619b271a321
